rpcd/playbooks/roles/rpc_maas/templates/container_storage_checks.yaml.j2

```diff
@@ -0,0 +1,20 @@
+{% set label = "container_storage_check" %}
+{% set check_name = label+'--'+ansible_hostname %}
+type        : agent.plugin
+label       : "{{ check_name }}"
+period      : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
+timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
+disabled    : "{{ (check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
+details     :
+    file    : run_plugin_in_venv.sh
+    args    : ["{{ maas_plugin_dir }}container_storage_check.py", "--thresh", "{{ percent_used_critical_threshold }}"]
+alarms      :
+    container_storage_percent_used_critical:
+        label                   : container_storage_percent_used--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        disabled                : {{ (('container_storage_percent_used_critical--' + ansible_hostname) | match(maas_excluded_alarms_regex)) | ternary('true', 'false') }}
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["container_storage_percent_used_critical"] >= {{ percent_used_critical_threshold }}) {
```

Comment:
It appears that the script returns 1 or 0 based on it passing the threshold.  Because percent_used_critical_threshold is equal to 85, I don't think it'll ever actually trigger.  Should it be if != 1 ?